query/parse.go

```diff
@@ -124,7 +125,11 @@ func parseExpr(in []byte) (Q, int, error) {
 		}
 		expr = &caseQ{text}
 	case tokRepo:
-		expr = &Repo{Pattern: text}
+		if text == "" {
+			return nil, 0, fmt.Errorf("the repo: expr must have an argument")
+		}
```

Comment:
`repo:` is a valid query, it matches everything. I think the empty string compiles as a regex? If not, maybe special case it to `.*`?